scripts/exregional_make_ics.sh

```diff
@@ -85,7 +85,7 @@ fi
 #-----------------------------------------------------------------------
 #
 if [ $RUN_ENVIR = "nco" ]; then
-    extrn_mdl_staging_dir="${DATAROOT}/get_extrn_ics.${share_pid}"
+    extrn_mdl_staging_dir="${DATAROOT}/aqm_get_extrn_ics_${cyc}.${share_pid}"
```

Comment:
Not necessary change if a new data directory is not used.

Reply:
This job is one of the four job require special DATA directory naming used as input of another job. 
Same explanation as above.